Scripts/build-archive.sh

```diff
@@ -27,6 +27,31 @@ if [ ! -d "$PRODUCTS_DIR/iOS/AppCenterDistributeResources.bundle" ] || \
   exit 1
 fi
 
+# Verify prefix for framework classes.
+framework_classes() {
+  nm -gjoUC "$1" | awk '{print $2}' | grep "_OBJC_CLASS_" | cut -d_ -f5-
+}
+verify_framework_prefixes() {
+  local name=${1##*/}
+  name=${name%.*}
+  local classes=$(framework_classes "$1/$name")
+  for prefix in ${@:2}; do
+    classes=$(echo "$classes" | grep -v $prefix)
+  done
+  echo "$classes"
+  [ -z "$classes" ]
+}
+for framework in $PRODUCTS_DIR/**/*.framework; do
+  invalid_prefix_classes+=($(verify_framework_prefixes "$framework" "MSAC" "MSPL")) || invalid_prefix_framework+=(${framework#"$PRODUCTS_DIR"/})
```

Comment:
Can we make "MSAC", "MSPL" constants? It will be easier to grasp for a reader (and modify, if god forbid we need that 😄 )

Reply:
I don't think that it makes sense in bash here